src/flb_output.c

```diff
@@ -539,6 +543,23 @@ int flb_output_set_property(struct flb_output_instance *ins,
         ins->tls_key_passwd = tmp;
     }
 #endif
+    else if (prop_key_check("storage.total_limit_size", k, len) == 0 && tmp) {
+        if (strcasecmp(tmp, "off") == 0 ||
+            strcasecmp(tmp, "false") == 0) {
+            /* no limit for filesystem storage */
+            limit = -1;
+            flb_info("[config] unlimited total_limit_size");
```

Comment:
the message is very generic, which plugin is triggering the message ?

Reply:
updated